mantatail.py

```diff
@@ -71,30 +70,21 @@ def recv_loop(self, user_info: Tuple[str, socket.socket]) -> None:
 
                     if user is None:
                         if verb_lower == "user":
-                            _user_name = message.split(" ", 1)[0]
+                            _user_name = message
                         elif verb_lower == "nick":
                             _nick = message
                         else:
-                            error_code, error_info = irc_responses.ERR_NOTREGISTERED
-                            user_socket.sendall(
-                                bytes(
-                                    f":mantatail {error_code} * {error_info}\r\n",
-                                    encoding="utf-8",
-                                )
-                            )
+                            (error_code, error_info) = irc_responses.ERR_NOTREGISTERED
+                            user_socket.sendall(bytes(f":mantatail {error_code} * {error_info}\r\n", encoding="utf-8"))
 
                         if _user_name and _nick:
                             user = User(user_host, user_socket, _user_name, _nick)
-                            command_handler: IrcCommandHandler = IrcCommandHandler(
-                                self, user
-                            )
+                            command_handler: IrcCommandHandler = IrcCommandHandler(self, user)
```

Comment:
Should work like this, after the code change in #23:

```suggestion
                            command_handler = IrcCommandHandler(self, user)
```